EX02/app/src/main/java/com/example/Game_of_throne/Data.java

```diff
@@ -0,0 +1,12 @@
+package com.example.Game_of_throne;
+
+    public class Data {
```

Comment:
It is best practice for classes to have intuitive names that can refer to the data/features they hold.
I think a better name here could've been "Character" or something